src/modules/token/fungible/__fixtures__/mockTokens.js

```diff
@@ -62,7 +62,7 @@ export const mockAppsTokens = {
       displayDenom: 'lisk',
       symbol: 'LSK',
       logo: {
-        png: 'https://downloads.lisk.com/lisk/images/tokens/lsk.png',
+        png: 'https://lisk-qa.ams3.digitaloceanspaces.com/Artboard%201%20copy%2019.png',
```

Comment:
probably not a good idea to refer to this png? this will be destroyed once development is completed.

Reply:
It would be good to have this fully mocked, Clemente asked service about this last week but it was not a priority. So for now, to be able to test this feature in mock mode, it feels better to have a working correctly displaying image than a dummy one which would not show lisk logo